package.json

```diff
@@ -1,50 +1,37 @@
 {
-    "name": "lwc-utils",
-    "version": "1.1.0",
-    "scripts": {
-        "lint": "prettier --write utils-core utils-recipes",
-        "reinstall": "npm ci"
-    },
-    "author": "tsalb",
-    "license": "BSD 3",
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/tsalb/lwc-utils"
-    },
-    "keywords": [
-        "lwc",
-        "lwc-utils",
-        "salesforce",
-        "util",
-        "datatable",
-        "lightning"
-    ],
-    "bugs": {
-        "url": "https://github.com/tsalb/lwc-utils/issues"
-    },
-    "homepage": "https://github.com/tsalb/lwc-utils#readme",
-    "devDependencies": {
-        "@salesforce/eslint-config-lwc": "^0.7.0",
-        "@salesforce/eslint-plugin-aura": "^1.4.0",
-        "eslint": "^7.8.0",
-        "eslint-config-prettier": "^6.11.0",
-        "husky": "^4.3.0",
-        "lint-staged": "^10.4.0",
-        "prettier": "^2.2.1",
-        "prettier-plugin-apex": "^1.7.0",
-        "rimraf": "^3.0.2"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
-    "lint-staged": {
-        "**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}": [
-            "prettier --write"
-        ],
-        "**/{aura|lwc}/**": [
-            "eslint"
-        ]
-    }
+  "name": "lwc-utils",
+  "version": "1.1.0",
+  "scripts": {
+    "eslint:validate": "eslint **/lwc/**/*.js",
+    "prettier:validate": "prettier --check '**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}'",
+    "prettier:fix": "prettier --write '**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}'",
+    "reinstall": "npm ci"
+  },
```

Comment:
White space changes when this file was pretty-fied make it hard to read. Note the husky and lint-staged blocks are removed. Also note that the scripts block has been updated.